modules/command/src/commands/help.ts

```diff
@@ -63,7 +64,7 @@ export const help: Command = async (message) => {
 				},
 			]
 		);
-		helpSummary.discord.embeds![0].fields!.push({
+		(helpSummary.discord.embeds as APIEmbed[])![0].fields!.push({
```

Comment:
Please remove this cast and update the type definitions

Reply:
Types are upto date it just freaks out without it.

Type for embeds right now is `(APIEmbed | JSONEncodable<APIEmbed>)[] | undefined`

Reply:
Hmm is that cast safe or is there some checks that can be done to achieve the same thing?

If you think it's save happy to merge

Reply:
It should be safe let me test it rq

Reply:
All the JSonEncoable BS does is add a `toJSON` function

Reply:
Yes but it could potentially not be an array which would cause errors. We are attempting to access the first index of the array so I beleive having code to ensure its an array would be good.

Reply:
Something like

If (embeds not an array) embeds = [embeds]

Psudocode ofc